packages/ppx_pbt/ppx_pbt.0.2.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+license: "MIT"
+synopsis: "PPX Rewriter for property based testing"
+
+homepage: "https://gitlab.com/vch9/ppx_pbt"
+bug-reports: "https://gitlab.com/vch9/ppx_pbt/-/issues"
+
+maintainer: [
+  "Valentin Chaboche <valentin.chaboche@nomadic-labs.com>"
+]
+
+authors: [
+ "Valentin Chaboche <valentin.chaboche@nomadic-labs.com>"
+]
+
+depends: [
+  "dune" {>= "2.8.0"}
+  "menhir" {>= "20210419"}
+  "ocaml" {>= "4.10.2"}
+  "qcheck" {>= "0.17"}
+  "metaquot" {>= "0.4"}
+  "data-encoding" {>= "0.4"}
+  "zarith" {>= "1.12"}
+  "qcheck-alcotest" {>= "0.17"}
+  "ppx_deriving_qcheck" {>= "0.1"}
+  "ppx_deriving" {>= "5.0"}
+]
+
+pin-depends: [
+  ["ppx_deriving_qcheck.dev" "git+https://github.com/vch9/ppx_deriving_qcheck.git#dev"]
+]
```

Comment:
pin-depends are not allowed in opam-repository. Could you release it first?

Reply:
Oh well, I'm closing this until I release it. Thank you